nengo/solvers.py

```diff
@@ -19,6 +19,11 @@
 logger = logging.getLogger(__name__)
 
 
+def _rmses(A, Y, X):
```

Comment:
Not a big deal, but I would prefer the order of arguments here to be A, X, Y or Y, A, X, probably the first since the system is typically written AX = Y.